lib/PuppeteerSharp/Page.cs

```diff
@@ -126,6 +244,13 @@ private Page(Session client, Target target, FrameTree frameTree, bool ignoreHTTP
 
         #region Public Methods
 
+        /// <summary>
+        /// Returns metrics
+        /// </summary>
+        /// <returns>metrics</returns>
```

Comment:
I think it would be good to follow the wording "Task which resolves to" a list of metrics" (across the class)